plugins/SegmentEditor/SegmentSelectorControl.php

```diff
@@ -54,8 +54,7 @@ public function __construct($idSite = false)
             if ($segment['category'] == $visitTitle
                 && ($segment['type'] == 'metric' && $segment['segment'] != 'visitIp')
             ) {
-                $metricsLabel = Piwik::translate('General_Metrics');
-                $metricsLabel[0] = Common::mb_strtolower($metricsLabel[0]);
+                $metricsLabel = Common::mb_strtolower(Piwik::translate('General_Metrics'));
```

Comment:
Sweet. You can call this a fix as well then to completely remove it :) 👍  Thought you meant to really fix it